package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-lambda",
-  "version": "0.7.1",
+  "version": "0.7.2",
```

Comment:
The idea is to bump this up with https://github.com/motdotla/node-lambda/pull/53 so there's no need for this change here.